sqlmesh/dbt/basemodel.py

```diff
@@ -253,11 +226,17 @@ def sqlmesh_model_kwargs(self, model_context: DbtContext) -> t.Dict[str, t.Any]:
             if field_val:
                 optional_kwargs[field] = field_val
 
+        audits: t.List[AuditReference] = [(test.name, {}) for test in self.tests]
+        dependencies = self.dependencies
+        for test in self.tests:
+            dependencies = dependencies.union(test.dependencies)
```

Comment:
So are we inheriting audit dependencies in the model? I thought we decided not to do this

Reply:
As far as I remember we decided to extend the Snapshot's `parents` but not model's `depends_on`.

Reply:
Correct. That is the next step. In this PR, all column-level tests are implemented by inheriting the dependencies, which matches dbt behavior. In the follow-up PR, this can be decoupled where makes sense.

Reply:
OK, this is not correct. Let's come up with a follow-up ASAP then.

Reply:
Removed multi-model audits for the time being.